crates/rapier_testbed2d-f64/Cargo.toml

```diff
@@ -67,31 +67,35 @@ indexmap = { version = "2", features = ["serde"] }
 
 # Dependencies for native only.
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-bevy = { version = "0.15", default-features = false, features = [
+bevy = { version = "0.16", default-features = false, features = [
     "bevy_asset",
     "bevy_winit",
     "bevy_window",
     "x11",
+    "wayland",
```

Comment:
This fixes experience for linux users but crashes ci, not worth having, or maybe only for dev-dependencies ? or upstream fix?